src/solvers/fdsbp_unstructured/fdsbp_2d.jl

```diff
@@ -104,7 +184,7 @@ end
 # surface contributions are added.
 function calc_surface_integral!(du, u, mesh::UnstructuredMesh2D,
                                 equations, surface_integral::SurfaceIntegralStrongForm,
-                                dg::DG, cache)
+                                dg::FDSBP, cache)
```

Comment:
Why this change? The strong form should also work for plain DGSEM, shouldn't it?

Reply:
I thought because this lived in one of the `fdsbp` files, I should change it for the sake of consistency. If this was wrong I can change it back.

Reply:
Yes, please. It should also work for DG - but we should maybe add an additional test for it.

Reply:
Done.